hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1463,7 +1463,7 @@ possible configurations would overwhelm and obscure the important.
   </property>
   <property>
     <name>dfs.client.read.shortcircuit</name>
-    <value>false</value>
+    <value></value>
```

Comment:
(can't comment on the proper line)

but I think we also need to remove `dfs.domain.socket.path` and its default value of 'none', as that will mask the real domain socket path from hdfs-site

Reply:
Yes, I ignored this because it exists in my hbase-site.xml . I'm going to change that.